pkgs/applications/networking/cluster/stern/default.nix

```diff
@@ -15,11 +15,18 @@ buildGoPackage rec {
 
   goDeps = ./deps.nix;
 
-   meta = with lib; {
-      description      = "Multi pod and container log tailing for Kubernetes";
-      homepage         = "https://github.com/wercker/stern";
-      license          = licenses.asl20;
-      maintainers      = with maintainers; [ mbode ];
-      platforms        = platforms.unix;
-    };
+  postInstall = ''
```

Comment:
In the cross compiling case we could use the the output of `buildPackages`.
However the fix for the meantime would be also:

```
postInstall = optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
  mkdir -p $bin/share/bash-completion/completions
  $bin/bin/stern --completion bash > $bin/share/bash-completion/completions/stern
  mkdir -p $bin/share/zsh/site-functions
  $bin/bin/stern --completion zsh > $bin/share/zsh/site-functions/_stern
'';
```

Reply:
Oh nice, that's what I should have done 23 days ago. Coming right up.